rs/pocket_ic_server/src/state_api/state.rs

```diff
@@ -587,29 +582,6 @@ impl ApiState {
             .await
         }
 
-        // converts an HTTP request to an HTTP/1.1 request required by icx-proxy
```

Comment:
@nikolay-komarevskiy I dropped this middleware as ic-gateway can hopefully deal with HTTP/2 requests